route/account/_id/_login-cookie.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+module.exports = (config) => {
+    var cookieName, cookieSettings;
+
+    /**
+     * Clears the login cookie
+     *
+     * @param {Object} req Request
+     * @param {Object} res Response
+     */
+    function clear(req, res) {
+        if (req.cookies[cookieName]) {
+            res.setCookie(cookieName, "");
+        }
+    }
+
+
+    /**
+     * Returns the login cookie value from the request.
+     *
+     * @param {Object} req Request
+     * @return {string} May be an empty string.
+     */
+    function get(req) {
+        return req.cookies[cookieName];
+    }
+
+
+    /**
+     * Refreshes the login cookie, allowing it to last longer in the browser.
+     *
+     * @param {Object} req Request
+     * @param {Object} res Response
+     * @throws {Error} No cookie value to refresh
```

Comment:
Agreed.

I have made sure that every `throw` that happens outside of a promise and inside of a function where we document is now documented.  I also checked and possibly removed `@throws` from the jsdoc when the `throw` happened within a promise chain.